plugins/modules/identity/ipa/ipa_user.py

```diff
@@ -117,26 +117,26 @@
     gidnumber: '100'
     homedirectory: /home/pinky
     ipa_host: ipa.example.com
-    ipa_user: admin
+    community.general.ipa_user: admin
```

Comment:
```suggestion
    ipa_user: admin
```
Happens three more times in this module.

Reply:
@felixfontein thanks for catching this!
fixed